### PR TITLE
Remove CCM's dependency: pkg/features

### PR DESCRIPTION
**Pull request by cici37**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

This PR is part of the issue [here](https://github.com/kubernetes/cloud-provider/issues/29). See [KEP-removing-in-tree-providers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20190125-removing-in-tree-providers.md) for more details.
This PR is to remove CCM's dependency pkg/features

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by mml** [comment]
Re

/lgtm

File: staging/src/k8s.io/apiserver/pkg/features/kube_features.go [hunk 1]
```diff
@@ -149,6 +149,12 @@ const (
 	//
 	// Allows label and field based indexes in apiserver watch cache to accelerate list operations.
 	SelectorIndex featuregate.Feature = "SelectorIndex"
+
+	// owner: @khenidak
+	// alpha: v1.15
+	//
+	// Enables ipv6 dual stack
+	IPv6DualStack featuregate.Feature = "IPv6DualStack"
```
Comment (andrewsykim) [inline staging/src/k8s.io/apiserver/pkg/features/kube_features.go:157]: I think a local copy in cmd/cloud-controller-manager/app/core.go makes more sense unless there are other consumers that could benefit from a copy here.

Comment (cici37) [reply staging/src/k8s.io/apiserver/pkg/features/kube_features.go:157]: The tradeoff I made is future maintenance consideration VS importing extra dep. What would you think?

**Review by andrewsykim — COMMENTED** [review]
/approve
/lgtm

File: cmd/cloud-controller-manager/app/core.go [hunk 2]
```diff
@@ -35,7 +35,11 @@ import (
 	netutils "k8s.io/utils/net"
 
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
-	kubefeatures "k8s.io/kubernetes/pkg/features"
+)
+
+const (
+	// IPv6DualStack is here to avoid having to import features pkg
```
Comment (andrewsykim) [inline cmd/cloud-controller-manager/app/core.go:41]:
```
// IPv6DualStack enables ipv6 dual stack feature
// Original copy from pkg/features/kube_features.go
```

**Review by andrewsykim — COMMENTED** [review]
minor comment on feature gate comment

**Review by andrewsykim — COMMENTED** [review]
/approve
/lgtm

**Comment by nilo19** [comment]
https://github.com/kubernetes/kubernetes/blob/master/cmd/cloud-controller-manager/app/options/options.go#L48
Is here also supposed to be modified?

**Pull request merged**